packages/extester/src/util/coverage.ts

```diff
@@ -0,0 +1,111 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
I am wondering how much discussable can be copying this file from Microsoft repo? is this okey with licensing?